python/cudf/udf_cpp/strings/include/cudf/strings/udf/managed_udf_string.cuh

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <cudf/strings/udf/udf_string.hpp>
+
+namespace cudf {
+namespace strings {
+namespace udf {
```

Comment:
I think we weakly prefer
```suggestion
namespace cudf::strings::udf {
```
for cases like this, where all definitions are in the innermost namespace.